docs/user-guide/archived-items.md

```diff
@@ -58,9 +58,9 @@ For more details on navigating web archives within ReplayWeb.page, see the [Repl
 
 While crawling, Browsertrix will output one or more WACZ files — the crawler aims to output files in consistently sized chunks, and each [crawler instance](workflow-setup.md#crawler-instances) will output separate WACZ files.
 
-The files tab lists the individually downloadable WACZ files that make up the archived item as well as their file sizes and backup status.
+The **WACZ Files** tab lists the individually downloadable WACZ files that make up the archived item as well as their file sizes and backup status.
```

Comment:
Using bold to denote page / tab names is cool and potentially something to add to the styleguide in the same section that using italics to denote actions is mentioned.